app/models/domain/BusinessIncomeSourcesSummary.scala

```diff
@@ -27,7 +29,11 @@ case class BusinessIncomeSourcesSummary(incomeSourceId: String,
                                         totalDeductions: Option[BigDecimal],
                                         accountingAdjustments: Option[BigDecimal],
                                         taxableProfit: BigDecimal,
-                                        taxableLoss: BigDecimal)
+                                        taxableLoss: BigDecimal) {
+  // TODO returnNetBusinessProfitForTaxPurposes logic is a placeholder, real value will be calculated in SASS-8626
+  def returnNetBusinessProfitForTaxPurposes(): BigDecimal = if (netLoss != 0) -netLoss else netProfit
```

Comment:
we don't need if, `-0 == 0`, -netLoss is fine assuming != 0 is what you need

Can we have getNetBusinesProfitForTaxPurpose instead of return? get is more common

Reply:
Business-wise, the value is called NetBusinessProfitForTaxPurposes and will be calculated from multiple values giving a positive value (profit) or negative value (loss).
As a placeholder we are using netLoss and netProfit (both are always positive BigDecimals) for stubbing purposes to test profit or loss on different businesses, with the opposite value (netProfit and netLoss respectively) set to zero.
So this is the method we'll use for now, but can change the name to `getNetBusinesProfitOrLossForTaxPurpose ` for clarity?

Reply:
sounds good, I missed it is netProfit after else